pkg/liqoctl/install/handler.go

```diff
@@ -59,53 +72,66 @@ func HandleInstallCommand(ctx context.Context, cmd *cobra.Command, baseCommand,
 
 	helmClient, err := initHelmClient(config, commonArgs)
 	if err != nil {
+		s.Fail("Error initializing installer: ", err)
 		return err
 	}
 
 	oldClusterName, err := installutils.GetOldClusterName(ctx, kubernetes.NewForConfigOrDie(config))
 	if err != nil {
+		s.Fail("Error initializing installer: ", err)
 		return err
 	}
+	s.Success("Installer initialized")
+
+	s, err = printer.Spinner.Start("Retrieving cluster configuration from cluster provider")
+	utilruntime.Must(err)
 
-	fmt.Printf("* Retrieving cluster configuration from cluster provider... 📜  \n")
 	err = providerInstance.PreValidateGenericCommandArguments(cmd.Flags())
 	if err != nil {
+		s.Fail("Error validating generic arguments: ", err)
 		return err
 	}
 
 	err = providerInstance.ValidateCommandArguments(cmd.Flags())
 	if err != nil {
+		s.Fail("Error validating command arguments: ", err)
 		return err
 	}
 
 	err = providerInstance.PostValidateGenericCommandArguments(oldClusterName)
 	if err != nil {
+		s.Fail("Error validating generic arguments: ", err)
 		return err
 	}
 
 	err = providerInstance.ExtractChartParameters(ctx, config, commonArgs)
 	if err != nil {
+		s.Fail("Error extracting chart parameters: ", err)
 		return err
 	}
+	s.Success("Chart parameters extracted")
 
 	if commonArgs.DumpValues {
-		fmt.Printf("* Generating values.yaml file with the Liqo chart parameters for your cluster... 💾 \n")
+		s, err = printer.Spinner.Start("Generating values.yaml file with the Liqo chart parameters for your cluster")
 	} else {
-		fmt.Printf("* Installing or Upgrading Liqo... (this may take few minutes) ⏳ \n")
+		s, err = printer.Spinner.Start("Installing or Upgrading Liqo... (this may take few minutes)")
 	}
+	utilruntime.Must(err)
+
 	err = installOrUpdate(ctx, helmClient, providerInstance, commonArgs)
 	if err != nil {
+		s.Fail("Error installing or upgrading Liqo: ", err)
 		return err
 	}
 
 	switch {
 	case !commonArgs.DumpValues && !commonArgs.DryRun:
-		fmt.Printf("* All Set! You can use Liqo now! 🚀\n")
+		s.Success("* All Set! You can use Liqo now! 🚀")
```

Comment:
```suggestion
		s.Success("All Set! You can use Liqo now! 🚀")
```

In my terminal, the emoji is also not displayed correctly.